parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -504,6 +504,5 @@ coreos:
 {{else}}
 runcmd:
 - set -x
-- timeout 10 apt-mark hold walinuxagent{{GetKubernetesMasterPreprovisionYaml}}
-- timeout 10 apt-mark unhold walinuxagent
+- {{GetKubernetesMasterPreprovisionYaml}}
```

Comment:
Remove `apt-mark hold|unhold walinuxagent` from cloud-init implementation as all cluster bootstrapping runtime implementation is in CSE.